chart/compass/charts/director/config.yaml

```diff
@@ -81,15 +81,44 @@ graphql:
     deleteAutomaticScenarioAssignmentForScenario: ["automatic_scenario_assignment:write"]
     deleteAutomaticScenarioAssignmentsForSelector: ["automatic_scenario_assignment:write"]
 
+  field:
+    application:
+      auths: ["application.auths:read"]
+      webhooks: ["application.webhooks:read"]
+    application_template:
+      webhooks: ["application_template.webhooks:read"]
+    bundle:
+      instance_auth: ["bundle.instance_auths:read"]
```

Comment:
Should we add this scope to the integration systems as for now they have "runtime:write" which is needed to request bundleInstanceAuth creation?

Reply:
Currently, there are no use cases where integration systems create instance auths, so we can skip them in the response here and forbid the creation in another PR.